src/cpp/session/resources/tutorial_resources/index.html

```diff
@@ -7,9 +7,23 @@
    </head>
 
    <body class="rstudio-tutorials" style="visibility: hidden;">
+
       <div class="rstudio-tutorials-container">
          #!tutorials#
       </div>
+
+      <div class="rstudio-tutorials-footer">
```

Comment:
Instead of a `div` use `<footer>`. Same reasons as for using `main`.

Alternatively can add `role="contentinfo"` to the `div` to achieve the same result.